tslint.ts

```diff
@@ -1,10 +1,74 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as coreModules from 'resolve/lib/core';
 import * as tslint from 'tslint';
 import * as tslintAirbnb from 'tslint-config-airbnb';
 const tslintReact = require('tslint-react');
 
 const rulesAirbnb = tslintAirbnb.rules;
 const rulesReact = tslintReact.rules;
 
+// prepare regex for node core modules for 'ordered-imports' rule
+const modules = Object.entries(coreModules as Record<string, boolean>).reduce(
+  (acc: string[], [module, enabled]) => {
+    enabled && !module.startsWith('_') && acc.push(module);
+    return acc;
+  },
+  []
+);
+const coreModulesRegex = '^(' + modules.join('|') + ')$';
```

Comment:
Are module names escaped at this point?

Reply:
they are not... but there aren't any atm which e.g. contain a dot